### ISSUE TITLE
Output mismatch of the SVTR model with Visual_Semantic_Alignment input

### ISSUE THREAD

**Issue by paulcho98** [SijieLiu518/SGENet#8]
Hi, I have been trying to run your code, but have been running into some problems.
In [sgenet.py](https://github.com/SijieLiu518/SGENet/blob/master/model/sgenet.py), the SVTR model is used to output the attention map and text logits. 
`attn_map, text_logits = self.position_prior(x_) # text_logits.size: [1, 25, 37]`

However, the PaddleOCR2Pytorch's SVTR model code does not inherently output two things. I want to ask the following:
1. How do you get the attention map that is used here, and **what are its dimensions**? 
2. Am I correct in assuming the output text_logits is the final result given from the original SVTR output ([last line in the PaddleOCR2Pytorch code](https://github.com/frotms/PaddleOCR2Pytorch/blob/main/pytorchocr/modeling/backbones/rec_svtrnet.py#L263))?
3. Are there any other modifications that I have to make to the code other than what I have referred to above?

I have tried extracting the attention map from the final block of the SVTR model from the link above, but had difficulty progressing due to the dimension difference with what is required in the [Visual_Semantic_Alignment class](https://github.com/SijieLiu518/SGENet/blob/master/model/Position_aware_module.py#L259). This is especially confusing because the sequence length (50) doesn't match the positional encodings of the image features. 

If you could provide any information regarding these issues, that would be great. 
Thanks in advance.

**Comment by w777-star** [comment]
你好，我也遇到同样的问题，请问您最后是否解决，可否告知一下，我目前替换了一下发现和作者原论文数值差距4个点左右，我现在怀疑就是这个问题造成的
